libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplateEngine.cs

```diff
@@ -216,5 +217,20 @@ private string GetOsPath(string ambigiousPath)
                 return ambigiousPath.Replace("\\", "/");
             }
         }
+
+        /// <summary>
+        /// Parse lg content to LGTemplate list.
```

Comment:
Please add into this comments that, this function will track down all the imports of the content